h/admin.py

```diff
@@ -16,6 +17,31 @@ def index(_):
     return {}
 
 
+@view.view_config(route_name='admin_features',
+                  request_method='GET',
+                  renderer='h:templates/admin/features.html.jinja2',
+                  permission='admin')
+def features_index(_):
+    return {"features": models.Feature.all()}
+
+
+@view.view_config(route_name='admin_features',
+                  request_method='POST',
+                  permission='admin')
+def features_save(request):
+    session.check_csrf_token(request)
```

Comment:
You can just use the `check_csrf=True` predicate on the view config. Not sure you need to test it at that point.